.github/workflows/rust.yml

```diff
@@ -37,6 +37,11 @@ jobs:
         run: |
           npm install -g markdownlint-cli
 
+      - name: Setup OCaml (because of ocaml-gen)
```

Comment:
I guess we should change that comment at some point, now it's because of ocaml-rs, but no biggy ^^